attributes/default.rb

```diff
@@ -51,3 +51,9 @@
   '0.5.2_windows_386'  => '2e866812de16f1a6138a0fd1eebc76143f1314826e3b52597a55ac510ae94be6',
   '0.5.2_web_ui'       => 'ad883aa52e1c0136ab1492bbcedad1210235f26d59719fb6de3ef6464f1ff3b1'
 }
+
+###
+# selinux
+###
+
+default['selinux']['state'] = 'permissive'
```

Comment:
I would not put a different cookbook's attribute here as it'll take awhile for someone to track this down. If we want to default the attribute value we should do it prior to the include in the default recipe so that it is explicit. 
```ruby
if node['os'] == 'linux'
  node.default['selinux']['state'] = 'permissive'
  include_recipe 'selinux::default'
end
```